### ISSUE TITLE
Improvements for ISC 2025!

### ISSUE THREAD

**Issue by danny-lloyd** [ncasuk/ncas-isc#198]
- [ ] Typos in xarray (e.g. panfas and coorfinates)
- [ ] Add `xarray.open_dataset` to the start of ex01a

**Comment by sadielbartholomew** [comment]
Good idea @danny-lloyd, thanks for setting this up. FYI I've added one to the list.

**Comment by sadielbartholomew** [comment]
OK just adding ad-hoc now, as we are all doing! Since the list is getting longer I am attempting to separate into some sub-lists via checkboxes in the second-level nested bullet, e.g. I've put one for the netCDF session/content specifically.
